packages/Webkul/Shop/src/Http/Controllers/Customer/GDPRController.php

```diff
@@ -29,6 +29,8 @@ public function __construct(
      */
     public function index()
     {
+        abort_if(! core()->getConfigData('general.gdpr.settings.enabled'), 404);
```

Comment:
Not only index we need to disable all the route, either add this to constructor or route group level middleware.